webapp/packages/core-settings/src/SettingsManager/SettingsManagerService.ts

```diff
@@ -19,28 +19,37 @@ interface ScopeSettingsItem<T = any> {
 
 @injectable()
 export class SettingsManagerService {
+  get activeSettings(): Map<string, ISettingDescriptionWithScope<any>> {
```

Comment:
seems like we can store the settings as Map at first.

so 
#settings: Map<ScopeSettingsItem>;

Reply:
this settings is dynamic we can't store them in Map